UnityProject/Assets/Scripts/HealthV2/Living/Respiration/RespiratorySystemBase.cs

```diff
@@ -100,83 +84,82 @@ private void MonitorSystem()
 			}
 		}
 
-		private bool Breathe(IGasMixContainer node)
+		public void GasExchange(GasMix atmos, ReagentMix blood, ReagentMix toProcess, bool takeFromBlood = false)
```

Comment:
the if feels weird. We only use one of the arguments of the function or the other based on that if. Wouldn't an overload make more sense here?